### DISCUSSION TITLE
How to delete an added model supplier

### DISCUSSION THREAD

**Discussion by weiqingtao** [langgenius/dify#12224] [Help]
How to delete an added model supplier

**Comment by zichen34** [comment]
If you don't want to use models from a specific model supplier, I think you can just remove the API key?

**Comment by 18790811859** [comment]
![image](https://github.com/user-attachments/assets/0f800e95-1c54-4127-a6b3-694e28a175ce)
![image](https://github.com/user-attachments/assets/969e0413-30bc-436d-b4e6-a96d14ae26b0)
这样？
